### PR TITLE
Feature/cu-hn6q3b create clip editor UI

### PR DESCRIPTION
**Pull request by JoanMarcBardes**
Editor resource clip 2

### CODE REVIEW

**Comment by Jordi-Gil** [comment]
Stop! Merge into develop not Master @JoanMarcBardes

**Review by FrancPS — CHANGES_REQUESTED** [review]
As Jordi says

File: Project/Source/Resources/ResourceClip.cpp [hunk 1]
```diff
@@ -59,10 +63,71 @@ void ResourceClip::Load() {
...
+	if (ImGui::InputText("##clip_name", nameClip, 100)) {
+		name = nameClip;
+	}	
+
+	ImGui::ResourceSlot<ResourceAnimation>("Animaton", &animationUID);
+
+	ImGui::Checkbox("Loop", &loop);
+
+	ImGui::InputInt("Begin Index", &beginIndex);
+	SetBeginIndex(beginIndex);
```
Comment (JaimeBea) [inline Project/Source/Resources/ResourceClip.cpp:117]: There is no need to change 'beginIndex' and 'endIndex' into signed integers. You can use ImGui::InputScalar for unsigned integers.

**Review by JaimeBea — APPROVED** [review]

**Review by FrancPS — APPROVED** [review]

**Comment by JoanMarcBardes** [comment]
@JaimeBea 
I tested this branch with feature/CU-jd4mpr-implement-resource-meta-files and works fine when saving the resource, so every thing is right.

**Pull request merged**